komodo-test-utils/src/test/resources/org/komodo/test/utils/test-local-repository-in-memory-config.json

```diff
@@ -6,13 +6,6 @@
     },
     "node-types" :
     [
-    	"config/dv.cnd",
-        "config/xmi.cnd",
-        "config/med.cnd",
-        "config/mmcore.cnd",
-        "config/relational.cnd",
-        "config/vdb.cnd",
-        "config/TeiidSql.cnd",
         "config/komodo.cnd"
```

Comment:
It was suggested that the teiid-modeshape project does not use relational, med, mmcore or xmi cnd files. If they are likely to get removed, do they not need to be added back here? Otherwise, where would komodo depend on for them?
It seems a little brittle to trust teiid-modeshape for such dependencies when they may be removed at a subsequent juncture since they are unusued.